2022/69-bruno-rust.md

```diff
@@ -0,0 +1,41 @@
+# Bruno Rocha:  Introduction to Rust Programming
+
+## Upcoming Events
+Join our Meetup group for more events!
+https://www.meetup.com/data-umbrella
+
+## Key Links
+- Transcript: https://github.com/data-umbrella/event-transcripts/blob/main/2022/ 
+- Meetup Event: https://www.meetup.com/data-umbrella/events/289308172/
```

Comment:
@Cristinamulas can you add in the file name here?

Reply:
@reshamas done!